### PR TITLE
Reinstate html transforms

### PR DESCRIPTION
**Pull request by i-like-robots**
Reinstates the XSLT XML to HTML transforms after some refactoring.

CC @andygnewman

### CODE REVIEW

File: server/models/article-v3.js [hunk 1]
```diff
@@ -123,8 +116,32 @@ function applyModel (articleV1, articleV2) {
...
+			.then(result => {
+				article.openingXML = result;
+				return xslt(article.openingXML, 'main');
+			})
+			.then(result => {
+				article.openingHTML = result;
+			})
+	);
+
+	resolutions = Promise.all(resolutions);
```
Comment (i-like-robots) [inline server/models/article-v3.js]: I tried to make this as readable as possible, not DRY. Ideas for more readability welcome!

File: server/models/article/body-v1.js [hunk 2]
```diff
@@ -12,8 +12,12 @@ function isSlideshow (article) {
 
 module.exports = (article) => {
 	let body = (article.item.body && article.item.body.body || '')
+		// Remove new lines
+		.replace(/\r?\n|\r/g, '')
+		// Remove extraneous whitespace
+		.replace(/\s{2,}/g, ' ')
 		// Remove all tags except paragraphs
-		.replace(/<(?!\s*\/?\s*p\b)[^>]*>/gi, '');
+		.replace(/<(?!\s*\/?\s*p\b)[^>]*>/gi, '')
```
Comment (i-like-robots) [inline server/models/article/body-v1.js]: Poor linting effort there

File: server/models/article-v3.js (hunk 1 above)
Comment (andygnewman) [inline server/models/article-v3.js]: You could do 
```
if (articleV2) {
    let resolutions = [];

    article.bodyXML && ...
    
    article.openingXML && ...

    resolutions = Promise.all(resolutions);
}
```

but doesn't really add much

Comment (matthew-andrews) [inline server/models/article-v3.js]: can be `const resolutions` 🙃

Comment (i-like-robots) [inline server/models/article-v3.js]: whut?

File: server/models/article-v3.js [hunk 3]
```diff
@@ -123,25 +111,53 @@ function applyModel (articleV1, articleV2) {
...
+							title: image.title,
+							description: image.description,
+							url: image.binaryUrl,
+							width: image.pixelWidth,
+							height: image.pixelHeight
+						};
+					}
+				})
+		);
+	}
```
Comment (i-like-robots) [inline server/models/article-v3.js]: I think this block can be cleaned up if we change the handling of `mainImage` but that's for another day =]

**Comment by andygnewman** [comment]
:+1:

**Pull request merged**